charts/datalayer-spacer/README.md

```diff
@@ -27,19 +27,18 @@ For full documentation please checkout [Datalayer Tech](https://datalayer.tech/d
 | spacer.env.DATALAYER_OPENFGA_REST_URL | string | `"http://datalayer-openfga.datalayer-openfga.svc.cluster.local:8080"` |  |
 | spacer.env.DATALAYER_OPENFGA_STORE_ID | string | `""` |  |
 | spacer.env.DATALAYER_RUNTIME_ENV | string | `"prod"` |  |
-| spacer.env.DATALAYER_RUN_HOST | string | `""` |  |
+| spacer.env.DATALAYER_RUN_URL | string | `""` |  |
 | spacer.env.DATALAYER_SMTP_HOST | string | `""` |  |
 | spacer.env.DATALAYER_SMTP_PASSWORD | string | `""` |  |
 | spacer.env.DATALAYER_SMTP_PORT | string | `""` |  |
 | spacer.env.DATALAYER_SMTP_USERNAME | string | `""` |  |
+| spacer.env.DATALAYER_SOLR_ZK_HOST | string | `"solr-datalayer-solrcloud-zookeeper-headless.datalayer-solr.svc.cluster.local"` |  |
 | spacer.env.DATALAYER_SOLR_ZK_HOST | string | `""` |  |
```

Comment:
This is duplication is strange. Should the doc tool not update this correctly?

Reply:
Thx @fcollonval  - fixed with https://github.com/datalayer/helm-charts/pull/67/commits/78d8cd9cf73d568c93ddf8964bbd7e9db9eb95df